### ISSUE TITLE
Alert does not include context

### ISSUE THREAD

**Issue by gdhbashton** [Graylog2/graylog-plugin-hipchat#3]
Hi - thanks for your plugin, I've been trying it out with Graylog 1.0 and a HipChat API v2. 'room token' I can get a message logged into the room, but it did not include the message which caused the alert :(

```Trigger alert when there are X more  less 
than  0 messages in the last 1  minutes and 
then wait at least 1 minutes until triggering a new alert. (grace period)
When sending an alert, include the last 1  messages of the stream evaluated for this alert condition.```

In the HipChat Web chat window, the alert appears as

`Graylog Callbacks 1:55 PM Stream <Varnish Reloads> alert: Stream had 1 messages in the last 1 minutes with trigger condition more than 0 messages. (Current grace time: 1 minutes)`

There's nothing in the `graylog-server.log` at this time to help debug :/

Cheers,
Gavin.

**Comment by gtaylor** [comment]
This would be super helpful, agree!

**Comment by joschi** [comment]
Fixed by #10.
